main.js

```diff
@@ -30,7 +42,33 @@ define(function (require, exports, module) {
         LiveDevServerManager.registerServer({ create: _createServer }, 9001);
 
         // Turn preview iFrame On
-        browser.browse();
+        browser.init();
     });
-    
+
+    // Next, we wait until the LiveDevelopment module is initialized
+    // so we can safely swap our transport and launcher modules for
+    // the defaults.
+    function _configureIframeForUse() {
+        // Set up our transport and plug it into live-dev
+        PostMessageTransport.setIframe(browser.getBrowserIframe());
+        LiveDevelopment.setTransport(PostMessageTransport);
+
+        // Set up our launcher in a similar manner
+        Launcher.configure({
+            browser: browser,
+            server: _server
+        });
+        LiveDevelopment.setLauncher(Launcher);
+    }
```

Comment:
Move lines 65-73 into the `_configureIframeForUse` function, so that it will never call that code out of order (i.e., we'll never open live dev until we've finished configuring it.